lib/pbkit/pbkit.c

```diff
@@ -2580,14 +2518,9 @@ void pb_erase_depth_stencil_buffer(int x, int y, int w, int h)
     y2=y+h;
     
     p=pb_begin();
-    pb_push(p++,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_HORIZ,2);     //sets rectangle coordinates
-    *(p++)=((x2-1)<<16)|x1;
-    *(p++)=((y2-1)<<16)|y1;
-    pb_push(p++,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_DEPTH,3);     //sets data used to fill in rectangle
-    *(p++)=0xffffff00;      //(depth<<8)|stencil
-    *(p++)=0;           //color
-    *(p++)=0x03;            //triggers the HW rectangle fill (only on D&S)
-    pb_end(p);
+    p=pb_push2(p,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_HORIZ,((x2-1)<<16)|x1,((y2-1)<<16)|y1); //sets rectangle coordinates
```

Comment:
2 different commands (`NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_HORIZ` and the vertical equvialent)

Reply:
Reverted. :+1: